### PR TITLE
Update readme for examples/imageclassification

### LINKED ISSUE #768: analytics-zoo/zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification：Readme need updates about "install from pip" / "download prebuilt package"
more details pls refer to #718

### PR DESCRIPTION
**Pull request by Deliadd**
fixed #768

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-
+## Prepare predict dataset
 Put your image data for prediction in one folder.
 
-3. Run the example
-
+## Run after pip install
+You can easily use the following commands to run this example:
 ```bash
+export SPARK_DRIVER_MEMORY=200g
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: @dding3 Do we need 200g in image classification?

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 2]
```diff
@@ -21,7 +22,13 @@ imagePath=... // image path
 
 ANALYTICS_ZOO_HOME=...
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: What's this?

Comment (hkvision) [reply zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: Oops..... This is Scala example and no pip install....

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 3]
```diff
@@ -21,7 +22,13 @@ imagePath=... // image path
...
 ZOO_JAR_PATH=${ANALYTICS_ZOO_HOME}/lib/analytics-zoo-bigdl_BIGDL_VERSION-spark_SPARK_VERSION-ZOO_VERSION-jar-with-dependencies.jar 
+```
+See [here](https://analytics-zoo.github.io/master/#PythonUserGuide/run/#run-after-pip-install) for more running guidance after pip install.
 
+## Run with prebuilt package
+```bash
+export SPARK_HOME=the root directory of Spark
+export ANALYTICS_ZOO_HOME=the folder where you extract the downloaded Analytics Zoo zip package
 spark-submit \
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: Use spark-shell-with-zoo.sh to rewrite. Can refer to textclassification

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 4]
```diff
@@ -32,3 +39,4 @@ spark-submit \
 --class com.intel.analytics.zoo.examples.imageclassification.Predict \
 ${ZOO_JAR_PATH} -f $imagePath --model $modelPath --partition 4 --topN 5
 ```
+See [here](https://analytics-zoo.github.io/master/#PythonUserGuide/run/#run-without-pip-install) for more running guidance without pip install.
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: Wrong

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md [hunk 5]
```diff
@@ -1,18 +1,19 @@
 ## Image Classification example
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: Change this to # ?

File: zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md (hunk 1 above)
Comment (dding3) [reply zoo/src/main/scala/com/intel/analytics/zoo/examples/imageclassification/README.md]: The example is for inference, I don't think we need 200g memory. 8g should be good enough. If we need a more accurate data, maybe we can test this.

**Pull request closed without merging**